webrtc.html

```diff
@@ -2504,6 +2504,30 @@
             </ol>
           </dd>
 
+          <dt>RTCRtpSender createRtpSender (DOMString kind)</dt>
+
+          <dd>
+            <p>Equivalent to
+            calling <a href="#dom-peerconnection-addtrack"><code>addTrack(null)</code></a>,
+            but the returned RtpSender is configured to send a track
+            of the given kind.<p>
+
+            <p>The value
+              of <a href="#dom-rtpsender-track"><code>RtpSender.track</code></a>
+              of the returned RtpSender will be null.  When the track
+              is set, it must have the same kind.</p>
+
+            <p>Session parameters will be negotiated exactly as
+              though <a href="#dom-peerconnection-addtrack"><code>addTrack</code></a>
+              were called with a track of the given kind, except that
+              when track-specific information is needed, suitable
+              values will be provided in their place such that the
+              RtpReceiver on the remote endpoint will be able to
+              receive the media once the track is set.  For examle, if
+              a track ID is needed, a unique value will be generated
+              in its place.</p>
+          </dd>
```

Comment:
Rather than examples, I think that we need to be more precise here.  I understand that that is hard, and I think that I'd be OK with deferring that to a follow-up instead of gating this PR on having the extra detail.

For instance, we should describe *precisely* how the track ID is generated, if indeed it is needed.

Codec negotiation is trickier.  I would think that a largely software-based implementation would choose to advertise the software codecs it supports.  But we need to address the issue of sources that have encoders. Personally, I have concluded that generic codecs are all we need to concern ourselves with, since we no longer have any good way of predicting where media will come from. That means that source-specific encoders are properly exceptional and that they won't get good support for these sorts of features.